src/components/form/field_text/index.d.ts

```diff
@@ -15,6 +15,7 @@ declare module '@elastic/eui' {
     inputRef?: (ref: HTMLInputElement) => void;
     fullWidth?: boolean;
     isLoading?: boolean;
+    prepend: React.ReactNode;
```

Comment:
`prepend` is optional

```suggestion
    prepend?: React.ReactNode;
```

Reply:
whoops, good call!